bin/c8.js

```diff
@@ -8,31 +8,48 @@ const waitTillPortOpen = require('wait-till-port-open')
 
 getPort().then(async port => {
   foreground(
-    ['node', `--inspect-brk=${port}`].concat(process.argv.slice(2)),
+    [process.execPath, `--inspect-brk=${port}`].concat(process.argv.slice(2)),
     (done) => {
       console.info('actually got here')
+      done()
     }
   )
   try {
     await waitTillPortOpen(port)
     const client = await CRI({port: port})
 
+    const initialPause = new Promise((resolve) => {
+      client.once('Debugger.paused', resolve)
```

Comment:
was not waiting on the `Debugger.paused` event the root cause of the issues I was seeing? Interesting that this worked for CommonJS files.

Reply:
> was not waiting on the `Debugger.paused` event the root cause of the issues I was seeing? 

Yes, I believe so.

> Interesting that this worked for CommonJS files.

I agree, with the minor amendment that if you rename `c.mjs` to `d.js` the same issue would be seen.